cloud/src/definer/auth.definer.js

```diff
@@ -17,7 +17,9 @@ Parse.Cloud.define('signup', (request, response) => new Promise((resolve, reject
   user.set('lastname', String(request.params.lastname));
   user.set('username', String(request.params.username));
   user.set('password', String(request.params.password));
-  user.set('email', String(request.params.email));
+  if (String(request.params.email) !== '') {
+    user.set('email', String(request.params.email));
```

Comment:
wait i'm confused what does this do? 

`If the email isn't blank, set the email to ... blank?` Unless I'm missing something?